createBoard.json

```diff
@@ -0,0 +1,35 @@
+{
+    "selectOrganization" : "div[class='el-input']",
+    "nameInput" : "input[name='name']",
+    "cancelButton" : "button[name='prev_btn']",
+    "nextButton" : "button[name='prev_btn']",
+    "xButton" : "button[name='close-new-board-modal-btn']",
+    "scrumRadioButton" : "span[name='type_scrum']",
+    "kanbanRadioButton" : "input[id='kanban-board-radio']",
+    "uploadFile" : "input[name='file']",
+
+    "boradSideMenu" : {
+        "productBacklog" : "a[href='/boards/947'] > svg[class='vs-c-site-sign vs-c-project-menu-icon']",
+        "stats" : "a[href='/boards/947/stats']",
+        "history" : "a[href='/boards/947/history']",
+        "labels" : "a[href='/boards/947/labels'] ",
+        "files" : "a[href='/boards/947/files'] ",
+        "documents" : "a[href='/boards/947/documents']",
+        "worklogs" : "a[href='/boards/947/worklogs']",
```

Comment:
Nije dobro da se staticno vezes za id board-a kroz href.. Imamo druge nacine kroz cypress da to uradimo, ali za pocetak pokusaj da ga uhvatis bez id (u ovom slucaju je id 947), jer cim napravis novi board ovaj href vise nece biti validan.